config/default/Configuration.h

```diff
@@ -1524,6 +1524,9 @@
 
   // Moves the nozzle to the initial position
   #define NOZZLE_CLEAN_GOBACK
+
+  // Safety feature for side mounted purge buckets and such expected to be used on tool change
+  //#define NOZZLE_CLEAN_NO_Z
```

Comment:
"Enable for a purge/clean station that's always at the gantry height (i.e., no Z move needed)."